lib/matplotlib/axis.py

```diff
@@ -2059,18 +2060,24 @@ def get_view_interval(self):
         'return the Interval instance for this axis view limits'
         return self.axes.viewLim.intervalx
 
-    def set_view_interval(self, vmin, vmax, ignore=False):
+    def set_view_interval(self, vmin, vmax, ignore=False, emit=True,
+                          auto=False):
         """
         If *ignore* is *False*, the order of vmin, vmax
         does not matter; the original axis orientation will
         be preserved. In addition, the view limits can be
-        expanded, but will not be reduced.  This method is
+        expanded, but will not be reduced.
```

Comment:
OK, while we are here, can we make these docstrings numpydoc compliant?  

This file has almost no compliant docstrings (apparently it hasn't attracted the attention of @timhoffm, *or* he already has a PR in that hasn't been merged.)  

As a random example of whats needed have a look at: https://github.com/matplotlib/matplotlib/blob/b6acd0402e6120693c0ff2f5cd345c208bde29f0/lib/matplotlib/figure.py#L1060

If you don't feel like doing this, its probably fine, but it'd help..

Reply:
Should I make the entire file have compliant docstrings, or just the methods that I changed?

Reply:
ha ha - go for the whole file if you want, but thats *totally* unfair.  Just the strings you changed :wink:

Reply:
plus, if you think thats too annoying, don't feel obliged.  Someone will hit the whole file soon I'm sure...

Reply:
Did someone call me? :wink:

@shaynair No seriously, only update the docstrings as part of this PR if you are touching them anyway. It simplifies the reviews if they are not too large and cover only one aspect. Also, keeping bugfixes and major docstring cleanups separate makes possible backports easier. That said, you are very welcome to open a separate PR for further docstring changes in the file.